### ISSUE TITLE
Date fields for chapter program info default to today's date

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#4738]
Found while testing #4724 

If the ChA does not select a date for the season start or event launch, the form defaults to the current date and that is stored both on the form and in the info the admin sees for the chapter. We should defaulting to a particular date rather than having the user select something, in order to ensure data accuracy. Perhaps add a "no event scheduled" option for the launch date field and default the start date to "no date yet selected" or something like that.

<img width="821" alt="Screen Shot 2024-05-20 at 10 45 49 AM" src="https://github.com/Iridescent-CM/technovation-app/assets/516764/44642c00-2b37-44a3-912a-f7256a899570">

**Comment by shaun-technovation** [comment]
A fix for this is on Preview. The dates on the form should default to blank dates now.

**Comment by dboyer** [comment]
Confirmed on preview that this no longer defaults to today's date on either the ChA dashboard or on the admin page.

When I added a date, it was visible in both places.

Just noting that the Launch Event question will need to either be optional or have a "no launch event" option. I think all other fields are required. I'll test that when we test what functionality needs to be in place to give access to the chapter admin activity.
